.github/workflows/initial_CI.yml

```diff
@@ -0,0 +1,42 @@
+name: Django CI
+
+on:
+#   push:
+#     branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 20
+      matrix:
+        python-version: [3.7]
```

Comment:
since you're only using and supporting asingle Python version you can probably remove the matrix setting and simplify the file.

Reply:
fixed